.python-version

```diff
@@ -0,0 +1 @@
+3.4.1
```

Comment:
@dethos Nothing to say regarding the version itself . About pinning: I favour pinning Python's runtime version (mostly for the reasons @lopesrb presented); I always favour explicit over implicit. Any reason in particular not to do it? Is it the "maintenance" overhead? I rather have that while making sure we're all using the same version.